docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -1,42 +1,40 @@
-* `iat`: issued at - the time at which the token was issued, as a Unix timestamp (e.g. `1516239022`).
-* `iss`: issuer - the Frontend API URL of your instance (e.g. `https://clerk.your-site.com` if production instance, `https://your-site.clerk.accounts.dev` if development instance).
-* `nbf`: not before - the time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** JWT template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
-* `sid`: session ID - the ID of the current session (e.g. `sess_123`).
-* `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
-* `act`: actor - will only be included if the user is impersonating another user. See [user impersonation](/docs/custom-flows/user-impersonation) for more information. 
+The default claims are:
+
+- `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application. For example: `https://example.com`. This claim could be omitted if, for privacy-related reasons, `Origin` is empty or null.
+- `exp`: expiration time - the time after which the token will expire, as a Unix timestamp. Determined using the **Token lifetime** JWT template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+- `iat`: issued at - the time at which the token was issued as a Unix timestamp. For example: `1516239022`.
+- `iss`: issuer - the Frontend API URL of your instance. For example: `https://clerk.your-site.com` for a production instance or `https://your-site.clerk.accounts.dev` for a development instance.
+- `nbf`: not before - the time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** JWT template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+- `sid`: session ID - the ID of the current session (e.g. `sess_123`).
+- `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
+- `act`: actor - will only be included if the user is impersonating another user. See [user impersonation](/docs/custom-flows/user-impersonation) for more information.
 
 The following claims are only included if the user is part of an organization:
 
-* `org_id`: organization ID - the ID of the currently active organization that the user belongs to.
-* `org_slug`: organization slug - the slug of the currently active organization that the user belongs to.
-* `org_role`: organization role - the role of the user in the currently active organization.
+- `org_id`: organization ID - the ID of the currently active organization that the user belongs to.
```

Comment:
```suggestion
- `org_id`: organization ID - the ID of the active organization that the user belongs to.
```
I don't think we need "currently" in any of these descriptions, unless there are claims that specifically give info about "previously active" organizations.